packages/vmcloak/PKGBUILD

```diff
@@ -2,45 +2,52 @@
 # See COPYING for license details.
 
 pkgname=vmcloak
-pkgver=912.13853ff
-pkgrel=2
-epoch=2
+pkgver=955.fae1f29
+pkgrel=1
+epoch=3
```

Comment:
Why do you increment epoch if pkgver is higher?

Reply:
Nevermind. I was working on different WS for this PKGBUILD. Now fixed.